packages/xrpl/test/models/utils.test.ts

```diff
@@ -46,6 +48,65 @@ describe('Models Utils', function () {
     })
   })
 
+  describe('parseAccountRootFlags', function () {
+    // eslint-disable-next-line complexity -- Simpler to list them all out at once.
+    it('all flags enabled', function () {
+      const accountRootFlags =
+        AccountRootFlags.lsfDefaultRipple |
+        AccountRootFlags.lsfDepositAuth |
+        AccountRootFlags.lsfDisableMaster |
+        AccountRootFlags.lsfDisallowXRP |
+        AccountRootFlags.lsfGlobalFreeze |
+        AccountRootFlags.lsfNoFreeze |
+        AccountRootFlags.lsfPasswordSpent |
+        AccountRootFlags.lsfRequireAuth |
+        AccountRootFlags.lsfRequireDestTag |
+        AccountRootFlags.lsfDisallowIncomingNFTokenOffer |
+        AccountRootFlags.lsfDisallowIncomingCheck |
+        AccountRootFlags.lsfDisallowIncomingPayChan |
+        AccountRootFlags.lsfDisallowIncomingTrustline |
+        AccountRootFlags.lsfAllowTrustLineClawback
+
+      const parsed = parseAccountRootFlags(accountRootFlags)
+
+      assert.isTrue(
+        parsed.lsfDefaultRipple &&
+          parsed.lsfDepositAuth &&
+          parsed.lsfDisableMaster &&
+          parsed.lsfDisallowXRP &&
+          parsed.lsfGlobalFreeze &&
+          parsed.lsfNoFreeze &&
+          parsed.lsfPasswordSpent &&
+          parsed.lsfRequireAuth &&
+          parsed.lsfRequireDestTag &&
+          parsed.lsfDisallowIncomingNFTokenOffer &&
+          parsed.lsfDisallowIncomingCheck &&
+          parsed.lsfDisallowIncomingPayChan &&
+          parsed.lsfDisallowIncomingTrustline &&
+          parsed.lsfAllowTrustLineClawback,
+      )
+    })
+
+    it('all flags not set', function () {
```

Comment:
My bad, this one is a better title :)

```suggestion
    it('no flags set', function () {
```